modules/features/filters/src/main/java/au/com/shiftyjelly/pocketcasts/playlists/edit/SmartPlaylistOptionsColumn.kt

```diff
@@ -0,0 +1,159 @@
+package au.com.shiftyjelly.pocketcasts.playlists.edit
+
+import androidx.annotation.DrawableRes
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.Icon
+import androidx.compose.material.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.Role
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.unit.dp
+import au.com.shiftyjelly.pocketcasts.compose.AppThemeWithBackground
+import au.com.shiftyjelly.pocketcasts.compose.components.TextH30
+import au.com.shiftyjelly.pocketcasts.compose.components.TextH50
+import au.com.shiftyjelly.pocketcasts.compose.preview.ThemePreviewParameterProvider
+import au.com.shiftyjelly.pocketcasts.compose.theme
+import au.com.shiftyjelly.pocketcasts.models.type.PlaylistEpisodeSortType
+import au.com.shiftyjelly.pocketcasts.ui.theme.Theme.ThemeType
+import au.com.shiftyjelly.pocketcasts.images.R as IR
+import au.com.shiftyjelly.pocketcasts.localization.R as LR
+
+@Composable
+internal fun SmartPlaylistOptionsColumn(
+    sortType: PlaylistEpisodeSortType,
+    hasEpisodes: Boolean,
+    onClickSelectAll: () -> Unit,
+    onClickSortBy: () -> Unit,
+    onClickDownloadAll: () -> Unit,
+    onClickChromecast: () -> Unit,
+    onClickOpenSettings: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        modifier = modifier,
+    ) {
+        if (hasEpisodes) {
+            OptionRow(
+                iconId = IR.drawable.ic_playlists_select_episode,
```

Comment:
This icon seems to have an alpha of 0.5, so the colour doesn't seem correct compared to the others. I believe this is an issue with the design as it's the same in Figma.

<img height="300" alt="Screenshot_20250811_114542" src="https://github.com/user-attachments/assets/b091d38c-f159-4382-9e31-277eab08cc86" />

Reply:
I'll mark it in Figma.